benchmark/scripts/Benchmark_Driver

```diff
@@ -572,7 +580,9 @@ class BenchmarkDoctor(object):
 
     @staticmethod
     def _reasonable_setup_time(measurements):
-        setup = min([result.setup for result in BenchmarkDoctor._select(measurements)])
+        setup = min(
+            [result.setup or 0 for result in BenchmarkDoctor._select(measurements)]
+        )
```

Comment:
`None` compares less than anything in Python 2. In Python 3 `None` is not comparable at all, so I needed to provide a reasonable default value. 0 seems okay, since this function only has effect if `setup` is greater than 200000.